.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
I recommend that you ignore the .vscode directory as a best practice since it's only a local setup for your code editor.